FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2BadgeActivity.kt

```diff
@@ -43,7 +43,7 @@ class V2BadgeActivity : V2DemoActivity() {
 
             Column(
                 Modifier
-                    .fillMaxHeight()
+                    .fillMaxWidth()
```

Comment:
update it to .fillMaxSize() as the Column should use all the real state to color it background properly.

Reply:
Done